vl-infotext.src.js

```diff
@@ -86,25 +86,33 @@ export class VlInfotext extends NativeVlElement(HTMLDivElement) {
         }
     }
 
-    __observeElement(element, callback) {
-        const observer = new ResizeObserver(callback);
-        observer.observe(element);
-		return observer;
+    __observeChildElement(callback) {
+        const observer = new MutationObserver(callback);
+        observer.observe(this.firstElementChild, { childList: true, subtree: true });
+        return observer;
     }
 
-    __observeValueElement(callback) {
-        return this.__observeElement(this.__valueElement, callback);
-	}
-
-    __observeTextElement(callback) {
-        return this.__observeElement(this.__textElement, callback);
-	}
+    __observeChildElementResize(callback) {
+        const observer = new ResizeObserver(callback);
+        observer.observe(this.firstElementChild);
+        return observer;
+    }
 
-    __processValueElement() {
-        vl.infotext.dress(this.__valueElement);
+    __processChildElementChange(records) {
+        records.forEach(record => {
+            const isChildListType = record.type == 'childList';
+            const hasOneAddedNode = record.addedNodes && record.addedNodes.length == 1;
+            const hasOneRemovedNode = record.removedNodes && record.removedNodes.length == 1;
+            const isTextContentChanged = hasOneAddedNode && hasOneRemovedNode && record.addedNodes[0].textContent != record.removedNodes[0].textContent;
+            if (isChildListType && hasOneAddedNode && hasOneRemovedNode && isTextContentChanged) {
+                vl.infotext.dress(this.__valueElement);
+                vl.infotext.dress(this.__textElement);
```

Comment:
Redressen staat hieronder ook nog eens in __processChildElementResize. Misschien best refactoren naar een aparte methode.